care/utils/assetintegration/base.py

```diff
@@ -14,7 +14,7 @@ def handle_action(self, action):
         pass
 
     def get_url(self, endpoint):
-        return "http://{}/{}".format(self.middleware_hostname, endpoint)
+        return "https://{}/{}".format(self.middleware_hostname, endpoint)
```

Comment:
Are you sure this would work in a local env?

Reply:
Maybe not, but using http only was giving me unexpected errors while testing with the dev camera.

Reply:
move it to an env variable then

Reply:
Instead of an environment variable, I added the variable to the asset metadata with it defaulting to https as it makes more sense to have the connection type set per asset.

Reply:
ideally, we would not want any data to be sent over non HTTPS in production (especially medical data)

Reply:
Added a check to prevent sending insecure requests in production mode.